ae_page_objects.gemspec

```diff
@@ -4,8 +4,7 @@ require 'ae_page_objects/version'
 Gem::Specification.new do |s|
   s.name                      = "ae_page_objects"
   s.version                   = AePageObjects::VERSION
-
-  s.required_ruby_version     = '>= 1.9.3'
+  s.required_ruby_version     = '>= 2.0.0'
```

Comment:
Please revert this change. Changing the required_ruby_version would necessitate a major version bump. Generally, we'd like to keep this gem available to most folks as possible.